crates/ibc/src/core/ics24_host/identifier.rs

```diff
@@ -165,15 +161,9 @@ impl Display for ChainId {
     }
 }
 
-impl From<ChainId> for tendermint::chain::Id {
+impl From<ChainId> for String {
     fn from(id: ChainId) -> Self {
-        tendermint::chain::Id::from_str(id.as_str()).unwrap()
-    }
-}
-
-impl From<tendermint::chain::Id> for ChainId {
-    fn from(id: tendermint::chain::Id) -> Self {
-        ChainId::from(id.to_string())
+        id.to_string()
     }
 }
```

Comment:
This conversation looks redundant imo. `to_string()` is straightforward enough if anyone needs it.

Reply:
have removed